std/internal/digest/sha_SSSE3.d

```diff
@@ -120,15 +121,11 @@ version(USE_SSSE3)
         private immutable string X_CONSTANT = "XMM13";
     }
 
-    /* The control words for the byte shuffle instruction. */
-    align(16) private immutable uint[4] bswap_shufb_ctl =
-    [
-        0x0001_0203, 0x0405_0607, 0x0809_0a0b, 0x0c0d_0e0f
-    ];
-
-    /* The round constants. */
-    align(16) private immutable uint[16] constants =
+    /* The control words for the byte shuffle instruction and the round constants. */
+    align(16) public immutable uint[20] constants =
```

Comment:
It's passed to the asm function from `std.digest.sha`, because it's not possible to access globals in PIC code from asm. That's the fix!